en/products/vpn/more/vpn-or-proxy.ftl

```diff
@@ -0,0 +1,58 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+### URL: https://www-dev.allizom.org/products/vpn/more/vpn-or-proxy/
+
+# HTML page title and main title
+vpn-or-proxy-the-difference-between = The difference between a VPN and a web proxy
+
+# HTML page description and intro
+vpn-or-proxy-vpns-and-proxies-are = VPNs and proxies are solutions for online privacy and security. Here’s how these protect you and how to choose the best option.
+
+vpn-or-proxy-virtual-private-networks = Virtual private networks (VPNs) and secure web proxies are solutions for better privacy and security online, but it can be confusing to figure out which one is right for you. Here’s a look at how these services protect you and how to choose the best option for when you’re online.
+vpn-or-proxy-stop-isps-from = Stop ISPs from spying on you
+
+# Variables
+#   $firefox (url) - https://www.mozilla.org/firefox/new/
+#   $etp (url) - https://blog.mozilla.org/products/firefox/control-trackers-with-firefox/
+#   $trackers (url) - https://blog.mozilla.org/internet-culture/mozilla-explains/what-is-a-web-tracker/
+vpn-or-proxy-when-you-use-firefox = When you <a href="{ $firefox }">use { -brand-name-firefox }</a>, <a href="{ $etp }">Enhanced Tracking Protection</a> automatically blocks many third party <a href="{ $trackers }">web trackers</a> from following you around the web. But here’s an interesting fact: your internet service provider (ISP) that you are paying for an internet connection — can still observe and track you.
+vpn-or-proxy-because-your-internet = Because your internet traffic moves to and from your devices (computer, phone, tv, tablet) through your ISP, they can see where you go online. An ISP can see what sites you visit, how long you’re on them, your location and information about your devices. An ISP may not know the specifics of what you did on those sites (like what you bought, searched for or read) thanks to encryption, but they could make inferences about you based on the sites that you visited.  That personal data can be used to create detailed profiles about you. Why would ISPs do that? In short: this data is valuable.
+vpn-or-proxy-isps-can-use = ISPs can use this information for their own ad targeting or for monetization opportunities that could include and sharing your information with third parties interested in data mining, marketing and targeted advertising, which means less privacy and more tracking. Browsing in private mode doesn’t prevent ISPs from seeing where you go online. But sending your web traffic through a web proxy or VPN can make it much harder.
```

Comment:
Remove *and* in `... monetization opportunities that could include and sharing...`